File: problems/star_wars_virtuel_queue/submissions/accepted/Star_wars_virtuel_queue.java
import java.util.Scanner;

public class Star_wars_virtuel_queue{
    
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int numInputs = scan.nextInt();
        scan.close();
        // rows
        for(int i = 1; i <= numInputs; i++){ 
            
            int j = numInputs;
            
            // loop to print (size - i) spaces
            while(j-- > i) {
                System.out.print("0");
            }
            // loop to print (i) #s
            while(j-- >= 0) {
                System.out.print("1");
            }
            
            System.out.println();
        }
        
    }   
    
}

File: problems/star_wars_virtuel_queue/submissions/accepted/star_wars_virtuel_queue.go
package main

import (
	"fmt"
)

func main() {
	var n int
	fmt.Scan(&n)
	for i := n; i > 0; i-- {
		for x := 1; x < i; x++ {
			fmt.Print("0")
		}
		for y := i; y <= n; y++ {
			fmt.Print("1")
		}
		fmt.Println()
	}
}
